packages/web-components/src/anchor/anchor.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { css } from '@microsoft/fast-element';
+import { styles as ButtonStyles } from '../button/button.styles.js';
+
+// Need to support icon hover styles
+export const styles = css`
+  ${ButtonStyles}
+
+  .content {
+    text-align: center;
```

Comment:
curious why this is different from button?

Reply:
We can add it to button if we want, but it doesn't make a bit of difference to the actual button code. I'm open here. I kept it separate to not make a pattern of hoisting deltas into a single stylesheet, especially if we're going to reuse. With the perf gate up we can look to identify if this is as performant or less than extracting the entire button styles to a css partial.

Reply:
Update here - given that compound button text *is not* centered, I don't think we want to add this to button as of yet.